functions/src/models/user.models.ts

```diff
@@ -20,7 +20,6 @@ export interface TanamUserInvited {
   email: string;
   invited: any;
```

Comment:
What value is held in the `invited` field? Put a type to it for clarity.

Reply:
Resolve on this commit https://github.com/oddbit/tanam/pull/257/commits/0822192a47e16d5da4de432c77f9bc6ef519a0a3